docs/content/theory.rst

```diff
@@ -14,11 +13,119 @@ Here's how it works.
     :height: 500px
     :align: center
 
-Matrix modifications
-=======
+User input 
+~~~~~~~~~~~~~~
+
+``timex_lca`` requires 3 inputs:
+ 
+1. a static foreground system model with
+2. temporal information using the attribute ``temporal_distribution`` on technosphere or biosphere exchanges in the foreground system modeel, and 
+3. a set of background databases, which must have a reference in time.`
+
+.. raw:: html
+
+    <details>
+        <summary>
+            <span style="color: white; padding: 4px; cursor: pointer;">ℹ️</span>
+            <strong style="margin-left: 8px;">More info on inputs</strong>
+        </summary>
+        <div style="background-color: #f0f7ff; padding: 10px;">
+            <ul>
+                <li>The foreground system must have exchanges linked to one of the background databases. These exchanges at the intersection between foreground and background databases will be relinked by <code>timex_lca</code>.</li>
+                <li>Temporal distributions can occur at technosphere and biosphere exchanges and can be given in various forms, see <a href="https://github.com/brightway-lca/bw_temporalis/tree/main">BW Temporalis documentation</a>, including absolute (e.g. 2024-03-18) or relative (e.g. 3 years before) nature and can have different temporal resolution (down to seconds but later aggregation supports resolutions down to hours).</li>
+                <li>Temporal distributions are optional. If none are provided, no delay between producing and consuming process is assumed and the timing of the consuming process is adopted also for the producing process.</li>
+            </ul>
+        </div>
+    </details>
```

Comment:
Why not use markdown syntax? Seems a bit complicated like this, but it doesn't really matter I guess as nobody's gonna see it :)